static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -238,8 +238,7 @@ function SidebarItem({
                   >
                     <LabelHook id={id}>
                       <TruncatedLabel>{label}</TruncatedLabel>
-                      {badges}
-                      {additionalContent}
+                      {additionalContent || badges}
```

Comment:
```suggestion
                      {additionalContent ?? badges}
```

this __probably__ is more correct. but I'm indifferent

Reply:
Sold!